### ISSUE TITLE
What do the outputs represent

### ISSUE THREAD

**Issue by haiqinzhong** [isyangshu/MambaMIL#3]
Dear isyangshu,
         I hope this message finds you well. I recently came across your repository MambaMIL on GitHub and found the work particularly interesting. However, I am having trouble understanding the meaning of the outputs in MambaMIL. Could you tell me what do the outputs(**hazards, S**) represent?
         Thank you very much for your time and assistance. I look forward to your response.

**Comment by wyhsleep** [comment]
Here, hazards and S specifically refer to the hazard function and the survival function, which are central concepts in survival analysis. Hazards values represent the instantaneous risk or hazard rate of an event (like death) occurring at each time point. In survival analysis, this is commonly referred to as the hazard function or hazard rate, describing the conditional probability of the event occurring in a very short interval, given that the event has not occurred up to that time point.
The survival function is based on the assumption that the survival probability at any given time point is the product of the survival probability at the previous time point and the probability that the event does not occur at the current time point. Thus, S provides the cumulative probability of a patient surviving through various time points, commonly used to assess how a patient’s survival chances diminish over time.

If you want to use the model to conduct WSI Classification, just using the logits from `logits = self.classifier(h)  # [B, n_classes]` for classification.

**Comment by haiqinzhong** [comment]
@wyhsleep thanks
